File: examples/base/14.py

@fields({'i':dyn})
class Inner:
	def set(self:Inner, x:dyn):
		self.i = x
	def get(self:Inner):
		return self.i


@fields({'i':Inner})
class Main:
	i = Inner()
	def main(self:Main) -> dyn:
		self.i = Inner()
		self.i.set(1)
		self.i.set("test")
		return self.i.get()

File: examples/strongscript/14.ts
class Inner {
  var i : any
  set(x:any) { i = x }
  get():any { return x }
}

class Main {
  main():any {
    i = new Inner()
    i.set(1)
    i.set("test")
    return i.get()
  }
}
